src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -799,13 +797,15 @@ private String getLastMovementTypeCode() {
     }
 
     public OffenderPhysicalAttributes getLatestPhysicalAttributes() {
-        return offenderPhysicalAttributes.stream().max(Comparator.comparing((pa) -> {
-            if (pa.getModifyDatetime() != null) {
-                return pa.getModifyDatetime();
-            } else {
-                return pa.getCreateDatetime();
-            }
-        })).orElse(null);
+        return offenderPhysicalAttributes.stream()
+            .sorted(Comparator.comparing((pa) -> pa.getId().getSequence()))
```

Comment:
get in the right order before doing max in case more than one attribute share the same modifyDateTime